docs/docs/prompts/optimize-prompt.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 5
+sidebar_label: Optimize Prompts
+---
+
+import { APILink } from "@site/src/components/APILink";
+
+# Optimize Prompts (Experimental)
+
+MLflow allows you to plug your prompts into advanced prompt optimization techniques through MLflow's unified interface using the <APILink fn="mlflow.genai.optimize_prompt" /> API.
+This feature helps you improve your prompts automatically by leveraging evaluation metrics and labeled data.
+Currently, DSPy's MIPROv2 algorithm is supported by this API.
```

Comment:
Could we link to dspy's documentation about this algorithm?

Reply:
Sounds good